debian/po/pt_BR.po

```diff
@@ -493,7 +490,7 @@ msgstr[1] ""
 "Renove sua assinatura em {url}"
```

Comment:
`ubuntu Pro` is lowercase in the translation, but `Ubuntu Pro` in the original text. Also, wasn't this translation meant to support singular and plural texts? Just wondering because both in the original string and in the translated one, it's plural (or at least using `(s)`).

Reply:
Also, sorry, just saw it now, but the translated string is using `PRO` instead of `Pro`.

Reply:
Good catch, I have solved the issue